build.sh

```diff
@@ -44,12 +44,6 @@ install_deps () {
     go get github.com/jgautheron/goconst/cmd/goconst
     go get honnef.co/go/tools/cmd/megacheck
     go get github.com/golang/lint/golint
-    install_golang_dep
-}
-
-install_golang_dep() {
-    go get github.com/golang/dep/cmd/dep
```

Comment:
What if a user (not CI) does not have dep installed? They'll just get an error?

Reply:
That's right. They will have to install dep on their own.

Reply:
Yeah, we should add it to the getting started instructions.

The problem is that we can't install dep for people without making assumptions or adding complexity. Similarly to how we can't install Go for people.